zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -296,6 +296,10 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
                     no_callee_code.expr(),
                     true.expr(),
                 );
+                cb.require_true(
+                    "Precompile addresses are always warm",
+                    and::expr([is_warm.expr(), is_warm_prev.expr()]),
+                );
```

Comment:
in begin tx we already set is_warm of precompiles to true, do you think we still need check here?

```
for addr in 1..=PRECOMPILE_COUNT {
            cb.account_access_list_write(tx_id.expr(), addr.expr(), 1.expr(), 0.expr(), None);
        } // rwc_delta += PRECOMPILE_COUNT
```